_data/projects/movie-quote-generator.yml

```diff
@@ -0,0 +1,10 @@
+name: Movie Quote Generator
+desc: A simple web app that generates random movie quotes. Built with ReactJs.
+site: https://kamiri-charles.github.io/movie-quote-generator
+tags:
+  - reactjs
+  - webdesign
+  - javascript
+upforgrabs:
+  name: help wanted
+  link: https://github.com/kamiri-charles/movie-quote-generator/issues
```

Comment:
Either change the name above or make this link as help wanted one @kamiri-charles

Reply:
Is this good??... @ritwik12

Reply:
Currently all of the open issues in this repository are not labelled:

<img width="551" src="https://user-images.githubusercontent.com/359239/197285397-1535c94a-8352-4765-b738-25216ef8ef24.png">

If you wish to use the `help wanted` label simply update it to this when you have some issues labelled:

```suggestion
  link: https://github.com/kamiri-charles/movie-quote-generator/labels/help%20wanted
```